src/app/error-boundary/ErrorBoundary.css

```diff
@@ -0,0 +1,3 @@
+.hideErrorOverlay > iframe {
```

Comment:
Do we have a more specific way to hide this. Is there an attribute or class in the iframe?

Reply:
No, the iframe does not have any class name and the only attribute is inline styles.

I tried simply removing the iframe, but the `getDerivedStateFromProps` and `render` functions of `ErrorBoundary` run before the iframe is appended to the body, so that doesn't seem like a viable option.

Reply:
Lets make it `.hideErrorOverlay > iframe[style]` to be a little more specific.

Reply:
👍